.github/workflows/osx_win_python_builds.yml

```diff
@@ -123,11 +123,9 @@ jobs:
           # the host system's "/" is mounted to "/host" in the manylinux container
           CIBW_ENVIRONMENT: >
             CCACHE_DIR=/host/home/runner/.cache/ccache
-            VALHALLA_RELEASE_PKG=${{ env.VALHALLA_RELEASE_PKG }}
```

Comment:
this is being done by a cmake script now, which configures a pyproject.toml.in, which is where the pkg name is needed